tests/semanticdb/InstrumentTyper.expect.scala

```diff
@@ -21,6 +21,6 @@ class InstrumentTyper/*<<=example.InstrumentTyper#*/ { self/*<<=local0*/: AnyRef
     List/*=>>scala.package.List.*//*=>>scala.collection.IterableFactory#apply().*/()
   )
   type AnnotatedType/*<<=example.InstrumentTyper#AnnotatedType#*/ = Int/*=>>scala.Int#*/ @param
-  def singletonType/*<<=example.InstrumentTyper#singletonType().*/(x/*<<=example.InstrumentTyper#singletonType().(x)*/: Predef/*=>>scala.Predef.*/.type) = ???/*=>>scala.Predef.`???`().*/
-  final val clazzOf/*<<=example.InstrumentTyper#clazzOf.*/ = classOf/*=>>scala.Predef.classOf().*/[Option/*=>>scala.Option#*/[Int/*=>>scala.Int#*/]]
```

Comment:
This change looks like a regression, what was the motivation for this change? I believe there's a bug in semanticdb-scalac that causes `classOf` to not be handled

Reply:
I'd be happy to remove it, I had been trying to blackbox replicate the results of the existing test cases, because they appeared to disagree with the spec sometimes, for example ```scala.Predef.`???`().``` and not as the spec says: ```scala/Predef.`???`().```

Reply:
For reference, here is the test I was comparing to: https://github.com/scalameta/scalameta/blob/master/tests/jvm/src/test/resources/example/InstrumentTyper.scala

Reply:
The expect test output replaces `/` with `.` to fix syntax highlighting on GitHub. Package symbols should always have `/`, for example `scala/Predef.foo().`

Reply:
Ah yes thanks for pointing that out